freqtrade/freqtradebot.py

```diff
@@ -679,8 +695,10 @@ def execute_entry(
                 amount=amount,
                 is_open=True,
                 amount_requested=amount_requested,
-                fee_open=fee,
-                fee_close=fee,
+                fee_open=self.exchange.get_fee(symbol=pair, taker_or_maker=('maker' if open_maker else 'taker')),
```

Comment:
Please take a look at the exchange's `add_dry_order_fee()` method.

As pointed out before - doing this here is pointless, and the fee  assigned here will only have an effect _until the order fills_ - after which it's overwritten with the value we assign via "dry-run order" handling (see `exchange.add_dry_order_fee()`).


I'm not against this change in general (so we use the proper fee type) - but the fee assigned in this place is wrong.

Reply:
I don't know how to do it correctly, but when I look at my dry run results, they all have the maker fee when using a market order strategy, when they shouldn't

Reply:
well that'll not change with this PR :laughing: 
As said - apply THIS change (or something similar) to the method pointed out above - and it should work.